towhee/tests/test_util/resnet50_embedding/resnet50_embedding.yaml

```diff
@@ -31,7 +31,7 @@ operators:
             type: map
     -
         name: 'embedding_model'
-        function: 'mock_operators/pytorch_cnn_operator'
+        function: 'local/pytorch_cnn_operator'
```

Comment:
So we will go through all the `cache_paths` to find the operator right? So we need to make sure the path is in the `cache_paths` list. However we have fixed the workflow issue, so I wonder do we need this change to go through all the paths?

Reply:
The way we cache now requires that local operators be put into main cache. Doing this will assign a "local" as author so that is why I changed it up